src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,33 +1,60 @@
 package edu.kis.vh.nursery;
 
+/**
+ * Class of default type rhymer
+ */
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int STACK_CAPACITY = 12;
 
-	public int total = -1;
+	private static final int EMPTY_STACK_VALUE = -1;
 
+	private int[] numbers = new int[STACK_CAPACITY];
+
+	private int total = EMPTY_STACK_VALUE;

```

Comment:
Extracting this constant is done incorrectly. Should show during bridge refactoring.